doctor/views.py

```diff
@@ -348,3 +356,20 @@ def embed_text(request) -> Union[FileResponse, HttpResponse]:
             response = FileResponse(img)
             cleanup_form(form)
             return response
+
+
+def get_document_number(request) -> HttpResponse:
+    """Get PACER document number from PDF
+
+    :param request: The request object
+    :return: PACER document number
+    """
+
+    form = BaseFileForm(request.GET, request.FILES)
+    if not form.is_valid():
+        validation_message = form.errors.get_json_data()["__all__"][0]["message"]
+        return HttpResponse(validation_message, status=NOT_ACCEPTABLE)
```

Comment:
HTTP 406 isn't correct here. See: https://datatracker.ietf.org/doc/html/rfc7231.html#section-6.5.6, which says:

```
   The 406 (Not Acceptable) status code indicates that the target
   resource does not have a current representation that would be
   acceptable to the user agent, according to the proactive negotiation
   header fields received in the request, and the server
   is unwilling to supply a default representation.
```

In other words, that's what you send when the client says it can do HTML, but the URL only knows how to return JSON or JPEG or something. The server says, "Gah! I can't give you HTML! NOT ACCEPTABLE."

I only skimmed, but I think the other places in this file where 406 responses are used should also be fixed as part of this PR. It's worth it to get these things right. Most are probably 400 errors instead:

```
   The 400 (Bad Request) status code indicates that the server cannot or
   will not process the request due to something that is perceived to be
   a client error (e.g., malformed request syntax, invalid request
   message framing, or deceptive request routing).
```

Reply:
Great, yeah the `BAD_REQUEST` seems a better status code since the response in those cases is due to a client error when sending the request.

I only used `INTERNAL_SERVER_ERROR` instead of  `BAD_REQUEST here: `https://github.com/freelawproject/doctor/blob/b788ca498a8bbe40e43e145e7487c256ee85318b/doctor/views.py#L76

Because here if there's a problem is due to a different reason since the form is already validated.

Reply:
I assume you checked upstream in CL to make sure these changes didn't cause any problems?

Reply:
Yes, I've checked this and it works well on CL.

I was just concerned about this:
```
if not response.ok:
     continue
```
To check if the microservice response was successful is used the `ok` property but I've confirmed that this will continue working well since the new status code 400 is included within the `ok` evaluation since this property uses `raise_for_status()` so status codes between 400 and 600 (including 400) will raise an error.

I also update a test on CL that was comparing the previous 406 status code error.